fitnessllm_dataplatform/task_handler.py

```
@@ -161,0 +1,201 @@
+            uid: Uid for user found in firebase.
+            data_source: Data source to download from.
+
+        Raises:
+            KeyError: If required options or environment variables are missing.
+            ValueError: If data source is not supported.
+        """
+        if not self.initialized:
+            self._startUp(uid)
+
+        if data_source not in [member.value for member in FitnessLLMDataSource]:
+            raise ValueError(f"Unsupported data source: {data_source}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)